perception/autoware_ground_segmentation/src/scan_ground_filter/node.hpp

```diff
@@ -148,7 +148,7 @@ class ScanGroundFilterComponent : public autoware::pointcloud_preprocessor::Filt
   // conform to new API
   virtual void faster_filter(
```

Comment:
```suggestion
  void faster_filter(
```

https://results.pre-commit.ci/run/github/429651616/1732784681.SyFDBr8iSM2mWATMHeaVSg

> perception/autoware_ground_segmentation/src/scan_ground_filter/node.hpp:149:  "virtual" is redundant since function is already declared as "override"  [readability/inheritance] [4]

Reply:
Thank you for pointing this out. It has been corrected.